### DISCUSSION TITLE
How to run in azure

### DISCUSSION THREAD

**Discussion by Mergpijp** [minvws/nl-kat-coordination#55] [Ideas]
I got the open kat app working locally. Now I want to run the app in azure. I got the following link as starting point:
[Azure tutoriial](https://learn.microsoft.com/nl-nl/azure/container-instances/tutorial-docker-compose) Can I use the make cat command?
I found that using docker-compose up myself will make the application unusable. Any suggestions?

**Comment by Mergpijp** [comment]
So I need something under build in the yml file. Or I need something under push: in the makefile. But what exactly?

**Comment by underdarknl** [comment]
Dear @Mergpijp 
Running this in azure will give you various options. I'm guessing you are going to want to run each program as a container, for this you could probably use the prebuild docker containers. Around these you would probably need some configuration to bind them all together.
Lets start a discussion on how to best get this going.

**Reply by Mergpijp** [reply]
Hey @underdarknl ,

It was easier to just run a VM with ubuntu on it. I managed to get openkat working. Just a follow up question how do I enable the other reports plus the clearance level? They are greyed out in the web app.

**Reply by underdarknl** [reply]
Once you are past the onboarding you should be able to create a report on almost every object from its object detail page.
Setting clearance levels can also been done via this page under the clearance levels tab
